### PR TITLE
Fix printing duplicated pod priorityClassName

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

> /kind bug
> /kind cleanup

**What this PR does / why we need it**:

Fix printing duplicated pod priorityClassName

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/describe/versioned/describe.go [hunk 1]
```diff
@@ -670,6 +670,8 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		w.Write(LEVEL_0, "Namespace:\t%s\n", pod.Namespace)
 		if pod.Spec.Priority != nil {
 			w.Write(LEVEL_0, "Priority:\t%d\n", *pod.Spec.Priority)
+		}
+		if len(pod.Spec.PriorityClassName) > 0 {
 			w.Write(LEVEL_0, "PriorityClassName:\t%s\n", stringOrNone(pod.Spec.PriorityClassName))
```
Comment (liggitt) [inline pkg/kubectl/describe/versioned/describe.go]: prefer "Priority Class Name"

**Comment by liggitt** [comment]
one nit, then lgtm

File: pkg/kubectl/describe/versioned/describe.go (hunk 1 above)
Comment (cwdsuzhou) [reply pkg/kubectl/describe/versioned/describe.go]: The original one is `PriorityClassName`, I also think `Priority Class Name` is better. Done!

**Pull request merged**